src/amo/components/CollectionManager/index.js

```diff
@@ -37,7 +38,18 @@ import type { ReactRouterType } from 'core/types/router';
 
 import './styles.scss';
 
+export const ADDON_ADDED_STATE_PENDING: 'ADDON_ADDED_STATE_PENDING'
+  = 'ADDON_ADDED_STATE_PENDING';
+export const ADDON_ADDED_STATE_SUCCESS: 'ADDON_ADDED_STATE_SUCCESS'
+  = 'ADDON_ADDED_STATE_SUCCESS';
+
+export type AddonAddedStatusType = |
```

Comment:
So the constants have "state" in their names but here we have "status". Would it make sense to have `AddonAddedStateType`?